unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Tests/EventCardComponentViewTests.cs

```diff
@@ -342,15 +344,19 @@ public void CloseModalCorrectly()
         Assert.IsFalse(eventCardModalComponent.isVisible);
     }
 
-    [Test]
-    public void RaiseOnCloseActionTriggeredCorrectly()
+    [UnityTest]
+    public IEnumerator RaiseOnCloseActionTriggeredCorrectly()
     {
         // Arrange
         eventCardModalComponent.Show();
 
         // Act
         eventCardModalComponent.OnCloseActionTriggered(new DCLAction_Trigger());
 
+        yield return null;
+        yield return null;
+        yield return null;
```

Comment:
Why is this necessary? please add a comment or delete

Reply:
Added a comment in [this commit](https://github.com/decentraland/unity-renderer/pull/2345/commits/19b1c45cc2a793728c2587b6c6774ad28f16dcdc)